Dockerfile

```diff
@@ -48,6 +48,9 @@ RUN \
       rm -rf /var/cache/apk/* && \
       chmod +x /usr/bin/wkhtmltopdf && \
       apk add g++
+  RUN apk add --no-cache --update \
```

Comment:
@erickueen please dont add another RUN, that creates another layer in the docker image that we don't need.

Reply:
Or is there any reason you want this to be another layer?

Reply:
I agree. A previous RUN can be used